agent/src/agent/pipeline/config/jython_scripts/snmp.py

```diff
@@ -45,7 +40,7 @@ def main():
                 return
 
         batch = sdc.createBatch()
-        res = requests.get(sdc.userParams['SNMP_SOURCE_URL'], timeout=60)
+        res = requests.get(sdc.userParams['SNMP_SOURCE_URL'], timeout=45)
```

Comment:
shouldn't this timeout be the value from query_timeout parameter in the source?

Reply:
There are two places where timeouts used:

1. This requests.get
2. and the other is used during real SNMP command execution: 
https://github.com/anodot/daria/blob/fix-snmp-offset/agent/src/agent/data_extractor/snmp/snmp.py#L138

**query_timeout** parameter from the source used in the second

You think we should have timeouts as query_timeout parameter in both request?

Reply:
I think we can create a separate parameter for this timeout. It should be configurable in case when a user has many hosts to query and it takes a lot of time to process the data